src/common/components/paymentMethods/paymentMethodForm/paymentMethodForm.tpl.html

```diff
@@ -45,7 +45,7 @@
 
 <div class="tab-indiv cart-tab show" ng-if="$ctrl.paymentType === 'bankAccount'">
 
-  <bank-account-form payment-form-state="$ctrl.paymentFormState" on-payment-form-state-change="$ctrl.onPaymentFormStateChange({ $event: $event })" payment-method="$ctrl.paymentMethod" hide-bank-account-legal-statement="$ctrl.hideBankAccountLegalStatement"></bank-account-form>
+  <bank-account-form payment-form-state="$ctrl.paymentFormState" on-payment-form-state-change="$ctrl.onPaymentFormStateChange({ $event: $event })" payment-method="$ctrl.paymentMethod" hide-payment-type-options="$ctrl.hidePaymentTypeOptions"></bank-account-form>
```

Comment:
Seems like we might need to add the new property to the credit card form as well, since the default might be bank account form.

Reply:
I figured since `default-payment-type` already exists, this would be enough. It's more flexible and the code is simpler.

Reply:
Looks like I forgot to remove this line. It was left over from hiding the legal message. Thanks :)